B1/Index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Dự Án 1</title>
+	<meta charset="utf-8">
+	<link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+	<center>
+		<div class="contentTop">
```

Comment:
Check the format structure of code..  I see many tab using in the child elements.
For each element has the same position should make vertical position are the same
Check the whole files.